README-comments.md

```diff
@@ -0,0 +1,268 @@
+
+## Register with IBM Cloud
+To use the IBM Cloud Container Service you need to register with IBM Cloud. You will be able to create a free Kubernetes cluster with a single [Node](https://kubernetes.io/docs/concepts/architecture/nodes/), as long as you [provide some payment details](https://console.bluemix.net/account/billing). Presumably to put off opportunistic Bitcoin miners. If you already have a Lite account, then you will need to add payment details to unlock your free cluster. If not, then use the [Pay as you go option](https://www.ibm.com/cloud/pricing) from the start.
+
+## Create your free cluster
+IBM Cloud provides a single command line tool: `bx` for controlling the resources and services available to you. Using the Container Service, you can create a Kubernetes cluster through using `bx` but you may find it easier, at least the first time, to create your free cluster through the [web interface](https://console.bluemix.net/containers-kubernetes/clusters).
+
+A short time after you've initiated the cluster creation, the web interface shows instructions for downloading the `bx` CLI, installing the container-service plugin into it, logging into your account ... all the way to configuring `kubectl` with the cluster and starting a local proxy to show the Kubernetes Dashboard for the cluster you've created. Like this:
+
+![Kubernetes Dashboard](images/KubeDash.png)
+
+## Somewhere to put Docker images
+When you run you microservices on Kubernetes, Kubernetes deploys Docker containers made from Docker images which it retrieves from a Docker registry. Your IBM Cloud account has a private Docker registry associated with served up by the IBM Cloud Container Registry. To start with your private registry is empty.
+
+The [Container Registry Quick Start](https://console.bluemix.net/containers-kubernetes/registry/start) gives instructions for installing the container-registry plugin to the `bx` CLI and how to create a namespace which can receive images pushed to it. This will come in useful later when we use Lightbend Orchestration for Kubernetes to push the Chirper application images to the registry.
+
+## Developing Chirper with the IBM Cloud
+You may have already followed [the instructions for setting up the Chirper example](https://github.com/longshorej/lagom-java-chirper-tooling-example/blob/master/README.md) which includes instructions specific to Minikube. The set of prequisites and steps are similar for deploying to IBM Cloud:
+
+Install the following programs on your system:
+
+* [Docker](https://store.docker.com/search?type=edition&offering=community)
+* [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+* [SBT](http://www.scala-sbt.org/)
+* [reactive-cli](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/#install-the-cli)
```

Comment:
URL is now https://developer.lightbend.com/docs/lightbend-orchestration-kubernetes/latest/cli-installation.html

Reply:
thanks